File: 1. Two Sum/brute search.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    var result = [];
    outside:
    for(var i=0; i<nums.length; i++) {
        let search = target - nums[i];
        for(var j=i+1; j< nums.length; j++) {
            if(nums[j] == search) {
                result[0] = i;
                result[1] = j;
                break outside;
            }
        }
    }
    return result;
};

File: 1. Two Sum/brute search.py
class Solution:
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        for idx, e in enumerate(nums):
            searching = target - e
            for j in range(idx+1,len(nums)):
                if nums[j] == searching:
                    return [idx, j]